unittests/linuxliketests.py

```diff
@@ -1704,8 +1702,14 @@ def test_prelinking(self):
                            stderr=subprocess.DEVNULL,
                            text=True, timeout=1)
         obj_files = p.stdout.strip().split('\n')
-        self.assertEqual(len(obj_files), 1)
-        self.assertTrue(obj_files[0].endswith('-prelink.o'))
+        if is_osx():
+            # There are at least two -- one for the prelinked object,
+            # another for the symbol list
+            self.assertGreaterEqual(len(obj_files), 1)
```

Comment:
based on this comment shouldn't it be 2 not 1 here?

Reply:
but also, I suspect that this assertions is less relevant with the `any` test than with the indexed test, since this would cause a nice failure rather than an assertion failure if obj_files was empty. It's probably fine to just drop it.

Reply:
> based on this comment shouldn't it be 2 not 1 here?

Yes, but I'm not sure why it kept on failing on the next assertion; on my box there's always a `SYM_SORTED` object file preceding `prelinked.o`, hence why I tried to work around it.

I'll go with your suggestion and drop the check altogether.